todolist/app/constants/styleConst.js

```diff
@@ -0,0 +1,13 @@
+//Colors
+export const GREY = "#D6DBDF";
+export const BLUE = "#87CEEB";
+export const RED = "#CB4335";
+export const WHITE = "#FFF"
+export const BLACK = "#000"
+
+//Sizes
+export const HEIGHT_HEADER_FOOTER = 50;
+export const FONT_SIZE_TITLE = 18;
```

Comment:
you have to extract to a different file only if you are repeating this value in different files. If not, just extract as a const in the same styles.js before the stylesheet